guides/source/webpacker.md

```diff
@@ -0,0 +1,219 @@
+
+
+The installation also calls the `yarn` package manager, creates a `package.json` file with a basic set of packages listed, and uses Yarn to install these dependencies.
+
+### Integrating Frameworks with Webpacker
+
+Webpacker also contains support for many popular JavaScript frameworks and tools. Typically, these are installed either when the application is created with something like `rails new myapp --webpack=<framework_name>` or with a separate command line task, like `rails webpacker:install:<framework_name>`.
+
+These integrations typically install the set of NPM packages needed to get started with the framework or tool, a "hello world" page to show that it works, and any other webpack loaders or transformations needed to compile the tool. The supported frameworks and tools are:
+
+INFO. It's possible to install frameworks not included in this list. These are basic integrations of popular choices.
+
+|Framework         |Install command                     |Description                                       |
+|------------------|------------------------------------|--------------------------------------------------|
+|Angular           |`rails webpacker:install:angular`   |Sets up Angular and Typescript                    |
+|CoffeeScript      |`rails webpacker:install:coffee`    |Sets up CoffeeScript                              |
+|Elm               |`rails webpacker:install:elm`       |Sets up Elm                                       |
+|ERB               |`rails webpacker:install:erb`       |Sets up ERB support on your Javascript files      |
+|React             |`rails webpacker:install:react`     |Sets up ReactJS                                   |
+|Stimulus          |`rails webpacker:install:stimulus`  |Sets up StimulusJS                                |
+|Svelte            |`rails webpacker:install:svelte`    |Sets up Svelte JS                                 |
+|TypeScript        |`rails webpacker:install:typescript`|Sets up Typescript for your project using Babel's TypeScript support|
+|Vue               |`rails webpacker:install:vue`       |Sets up VueJS                                     |
+
+For More information about the existing integrations, see https://github.com/rails/webpacker/blob/master/docs/integrations.md.
+
+Usage
+-----
+### Using Webpacker for JavaScript
+
+With Webpacker installed, by default any JavaScript file in the `app/javascripts/packs` directory will get compiled to its own pack file.
+
+So if you have a file called `app/javascript/packs/application.js`, Webpacker will create a pack called `application`, and you can add it to your Rails application with the code `<%= javascript_pack_tag "application" %>`. With that in place, in development, Rails will re-compile the `application.js` file every time it changes and you you load a page that uses that pack. Typically, the file in the actual `packs` directory will be a manifest that mostly loads other files, but it can also have arbitrary JavaScript code.
```

Comment:
```suggestion
So if you have a file called `app/javascript/packs/application.js`, Webpacker will create a pack called `application`, and you can add it to your Rails application with the code `<%= javascript_pack_tag "application" %>`. With that in place, in development, Rails will re-compile the `application.js` file every time it changes and you load a page that uses that pack. Typically, the file in the actual `packs` directory will be a manifest that mostly loads other files, but it can also have arbitrary JavaScript code.
```